theory/DDrppi/countpairs_rp_pi_impl.c.src

```diff
@@ -45,83 +45,86 @@ countpairs_rp_pi_func_ptr_DOUBLE countpairs_rp_pi_driver_DOUBLE(const struct con
         return function;
     }
 
-
     /* Array of function pointers */
     countpairs_rp_pi_func_ptr_DOUBLE allfunctions[] = {
-#ifdef __AVX512F__
-      countpairs_rp_pi_avx512_intrinsics_DOUBLE,
+        countpairs_rp_pi_fallback_DOUBLE,
```

Comment:
Is the whitespace correct in this line (since there is a change from the previous alignment)?

Reply:
Yep, everything is lined up and indented 4 spaces!